eo-maven-plugin/src/main/java/org/eolang/maven/ResolveMojo.java

```diff
@@ -202,6 +203,27 @@ private Collection<Dependency> deps() throws IOException {
             .collect(Collectors.toList());
     }
 
+    /**
+     * Check if runtime dependency is absent.
+     * @param deps Dependencies
+     * @todo #1361:90min Hardcoded version of EoRuntimeDependency.
+     *  See the EoRuntimeDependency constructor for more info.
+     *  It's much better to determine the version of the runtime library
+     *  dynamically. For example, we can fetch the latest version by http
+     *  or from config files.
+     */
+    private static void addRuntimeDependency(final Collection<Dependency> deps) {
+        final RuntimeDependencyEquality runtime = new RuntimeDependencyEquality();
```

Comment:
@volodya-lombrozo check this out: https://www.yegor256.com/2015/09/01/redundant-variables-are-evil.html

Reply:
Fixed